tools/lms/lms_siglen

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+#
```

Comment:
Do we need GPL boiler plate here? I don't know the answer so I'm asking.

Reply:
Not sure. The shell scripts in `tools/scripts/` don't have GPL, so was using that as ref. But `lms_siglen` should probably have `.sh` suffix like them.

Reply:
I originally experimented with putting the sig len calculation logic in makefile scripts but was too messy. And even if sig len is automatically set, the user will likely till need to manually config the header length for their platform. Hence I settled on this helper script.